src/main/resources/org/jenkinsci/plugins/docker/traceability/DockerTraceabilityPluginConfiguration/config.properties

```diff
@@ -0,0 +1,2 @@
+createImageFingerprints.title=Create image fingerprints on-demand
+showRootAction.title=Show Docker Traceability action on the main side panel
```

Comment:
Better than nothing, but still a very weird thing to be asking users to configure. We need to either find a way to determine mechanically whether this functionality is needed; or come up a with general UI landing site for “environmental information” that does not clog the main sidepanel (CC @gusreiber).